packages/create-playwright/assets/playwright.config.js

```diff
@@ -21,9 +21,7 @@ const config = {
     {
       name: 'Desktop Chromium',
       use: {
-        browserName: 'chromium',
-        // Test against Chrome Beta channel.
-        channel: 'chrome-beta',
+        ...devices['Desktop Chrome'],
```

Comment:
using chrome-* or normal Chrome will reduce the barrier of entry, since on e.g. on Codespaces this is not installed by default. If we would invoke it all the time (npx playwright install chrome) the browser would also get killed which is not a great user experience.